multiraft/multiraft_test.go

```diff
@@ -143,9 +142,7 @@ func (c *testCluster) createGroup(groupID roachpb.RangeID, firstNode, numReplica
 // the given node will win the election. Unlike elect(), triggerElection() does not
 // wait for the election to resolve.
 func (c *testCluster) triggerElection(nodeIndex int, groupID roachpb.RangeID) {
-	if err := c.nodes[nodeIndex].multiNode.Campaign(context.Background(), uint64(groupID)); err != nil {
-		c.t.Fatal(err)
-	}
+	c.nodes[nodeIndex].Campaign(groupID)
```

Comment:
In #3160 I found that `heartbeat_test.go` became unreliable when I made this change. Is it working OK for you?